renderer/brave_content_renderer_client.cc

```diff
@@ -32,7 +32,7 @@
 #endif
 
 #if BUILDFLAG(IS_ANDROID)
-#include "brave/components/brave_vpn/brave_vpn_utils.h"
+#include "brave/components/brave_vpn/common/brave_vpn_utils.h"
```

Comment:
brave_vpn_utils should not be guarded by IS_ANDROID, but both these files need ENABLE_BRAVE_VPN

Reply:
done